### PR TITLE
5540 create interesting file artifacts in batches

### PR DESCRIPTION
**Pull request by jkho**

### CODE REVIEW

**Comment by rcordovano** [comment]
@jkho, this PR has merge conflicts. Please resolve them, thanks!

**Comment by APriestman** [comment]
The merge seems to have broken some things.
C:\cygwin\home\apriestman\Work\autopsy\autopsy\Core\src\org\sleuthkit\autopsy\logicalimager\dsp\AddLogicalImageTask.java:435: error: cannot find symbol
                    postArtifacts(artifacts);
  symbol:   variable artifacts
  location: class AddLogicalImageTask
C:\cygwin\home\apriestman\Work\autopsy\autopsy\Core\src\org\sleuthkit\autopsy\logicalimager\dsp\AddLogicalImageTask.java:436: error: cannot find symbol
                    artifacts.clear();
  symbol:   variable artifacts
  location: class AddLogicalImageTask
C:\cygwin\home\apriestman\Work\autopsy\autopsy\Core\src\org\sleuthkit\autopsy\logicalimager\dsp\AddLogicalImageTask.java:454: error: cannot find symbol
            postArtifacts(artifacts);
  symbol:   variable artifacts
  location: class AddLogicalImageTask

**Comment by jkho** [comment]
merge was very confused. Fixing.

File: Core/src/org/sleuthkit/autopsy/logicalimager/dsp/AddLogicalImageTask.java [hunk 1]
```diff
@@ -365,21 +367,19 @@ private void addInterestingFiles(Map<String, List<Long>> interestingFileMap) thr
 
             List<Long> fileIds = entry.getValue();
             for (Long fileId: fileIds) {
```
Comment (APriestman) [inline Core/src/org/sleuthkit/autopsy/logicalimager/dsp/AddLogicalImageTask.java]: You need a cancellation check in this loop.

**Pull request merged**